torch/csrc/jit/script/python_sugared_value.h

```diff
@@ -19,6 +19,12 @@ inline std::shared_ptr<SugaredValue> toSimple(Value* v) {
   return std::make_shared<SimpleValue>(v);
 }
 
+// entry point for instantiating a SugaredValue from an IValue
```

Comment:
This isn't used outside of python_sugared_value.cpp, so it shouldn't be exposed.